apps/bfd-model/bfd-model-dsl-codegen/bfd-model-dsl-codegen-plugin/src/test/resources/GenerateTransformersFromDslMojoIT/mappings/FissClaim.yaml

```diff
@@ -0,0 +1,144 @@
+mappings:
+  - id: FissClaim
+    messageClassName: gov.cms.mpsm.rda.v1.fiss.FissClaim
+    entityClassName: gov.cms.test.FissClaim
+    transformerClassName: gov.cms.test.FissClaimTransformer
+    table:
+      name: FissClaims
+      schema: pre_adj
+      primaryKeyColumns:
+        - dcn
+      columns:
+        - name: dcn
+          sqlType: varchar(23)
+          nullable: false
+        - name: sequenceNumber
+          sqlType: bigint
+          nullable: false
+        - name: currStatus
+          sqlType: varchar(1)
+          javaType: char
+          nullable: false
+        - name: provStateCd
+          sqlType: varchar(2)
+        - name: medaProvId
+          sqlType: varchar(13)
+          comment: |
+            Medicare Provider ID:
+            
+            <p>The Medicare Provider ID consists of the following:
+            
+            <ul>
+              <li>Provider State Code
+              <li>Provider Type Facility Code
+              <li>Provider Emergency Indicator
+              <li>Provider Department Identification
+            </ul>
+        - name: medaProv_6
+          sqlType: varchar(6)
+        - name: totalChargeAmount
+          sqlType: decimal(11,2)
+        - name: receivedDate
+          sqlType: date
+        - name: lastUpdated
+          sqlType: timestamp with time zone
+        - name: pracLocAddr1
+          sqlType: varchar(max)
+        - name: lobCd
+          sqlType: varchar(1)
+        - name: servTypeCdMapping
+          enumType: ServTypeCdMapping
+          sqlType: varchar(20)
+        - name: servTypeCd
+          sqlType: varchar(1)
+        - name: freqCd
+          sqlType: varchar(1)
+        - name: apiSource
+          sqlType: varchar(24)
+          comment: "String specifying the source of the data contained in this record. Generally this will be the version string returned by the RDA API server but when populating data from mock server it will also include information about the mode the server was running in."
```

Comment:
YAML allows a string to be split across multiple lines as in the comment for `medaProvId`.  The script I used to generate this test mapping preserved the comments from the proto files exactly as is.

Reply:
Right, what I meant was that you could break it up on multiple lines for readability in the yaml, and then if you use a markdown, a single new line is ignored, so it would still transcript it as a single line to the destination.  Just makes the yaml more readable.